compiler/passes/convert-uast.cpp

```diff
@@ -1140,6 +1140,17 @@ struct Converter {
         return new UnresolvedSymExpr("BitwiseOrReduceScanOp");
       if (name == USTR("^"))
         return new UnresolvedSymExpr("BitwiseXorReduceScanOp");
+
+      if (name.str() == "max")
```

Comment:
Let's go ahead and use `USTR("max")` etc for these. You just have to add to compiler/dyno/include/chpl/framework/all-global-strings.h . I don't think we should be shy about adding these.

Reply:
Done 👍